src/App.css

```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
+
+* {
+  font-family: "Montserrat", sans-serif;
+}
+
+body {
+  background-color: #f5f6fa;
+}
+
+.app {
+  margin: 4%;
+}
```

Comment:
- [ ] This margin is unnecessary. Kindly remove this margin. This way, everything will fit into the viewport.